Unreal/CMakeLists.txt

```diff
@@ -173,6 +186,10 @@ else ()
     CARLA_UE_BUILD_COMMAND_PREFIX
       ${CARLA_UNREAL_ENGINE_PATH}/Engine/Build/BatchFiles/${UE_SYSTEM_NAME}/Build.sh
   )
+  set (
+    CARLA_UE_UAT_COMMAND_PREFIX
+      ${CARLA_UNREAL_ENGINE_PATH}/Engine/Build/BatchFiles/${UE_SYSTEM_NAME}/Build.sh
```

Comment:
```suggestion
      ${CARLA_UNREAL_ENGINE_PATH}/Engine/Build/BatchFiles/RunUAT.sh
```
As suggested in the office UE doc:
https://dev.epicgames.com/documentation/en-us/unreal-engine/build-operations-cooking-packaging-deploying-and-running-projects-in-unreal-engine#commandline

Reply:
This fixes the target platform error, but seems CARLA is not build as I get the next error:
```
LogPluginManager: Error: Plugin 'Carla' failed to load because module 'Carla' could not be found.  Please ensure the plugin is properly installed, otherwise consider disabling the plugin for this project.
```

I can try to open the Shipping editor but I get next error:
![Screenshot from 2024-03-31 10-56-54](https://github.com/carla-simulator/carla/assets/20727166/b510b318-0958-4947-88b2-282bca27c906)

Anyway I will commit it because now package generation is working, but seems we missed some configuration.